bitrise.yml

```diff
@@ -1419,14 +1419,34 @@ workflows:
             curl -o /tmp/key-file.json $BITRISEIO_GOOGLE_APPLICATION_CREDENTIALS_URL 
             $HOME/google-cloud-sdk/bin/gcloud version
             $HOME/google-cloud-sdk/bin/gcloud auth activate-service-account -q --key-file /tmp/key-file.json
-            $HOME/google-cloud-sdk/bin/gcloud config set project moz-firefox-ios-230118
+            $HOME/google-cloud-sdk/bin/gcloud config set project "$GOOGLE_CLOUD_PROJECT"
             $HOME/google-cloud-sdk/bin/gcloud beta firebase test ios run \
             --type robo \
             --app "$BITRISE_IPA_PATH" \
             --device=model=iphone13pro,version=15.7 \
             --results-bucket=firefox_ios_test_artifacts \
             --no-record-video \
             --client-details=matrixLabel="Bitrise"
+    - slack@3.1:
```

Comment:
Have you tried a more recent version of the `slack` Bitrise step? The latest one is `3.5.0`. I may try to upgrade some Bitrise steps in the near future.